app/apps/handlers.js

```diff
@@ -102,15 +102,25 @@ exports.list = (req, res, next) => {
 
 
 exports.details = (req, res, next) => {
+  let app;
+  let buckets;
+  let users;
+
   Promise.all([App.get(req.params.id), Bucket.list(), User.list()])
-    .then(([app, buckets, users]) => {
-      const current_user_is_app_admin = app.has_admin(req.user.auth0_id);
+    .then(([returned_app, returned_buckets, returned_users]) => {
+      app = returned_app;
+      buckets = returned_buckets;
+      users = returned_users;
```

Comment:
No need to change this, but it would be more concise (although perhaps not better) to do this like:
```javascript
.then((results) => {
  [app, buckets, users] = results;
```

Reply:
Hmmm. I like it better though. I should try to remember about deconstruction more.